addons/l10n_ph/models/account_tax.py

```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    l10n_ph_description = fields.Char("Philippines Description")
```

Comment:
What is the point of this field ?
There is already a description field: https://github.com/odoo/odoo/blob/14.0/addons/account/models/chart_template.py#L917
Why make a copy ?

Reply:
I asked JGR and she said this is required.

Reply:
But below in `_get_tax_vals` you use the field `description` to fill `l10n_ph_description` so it is just a duplicate of the original description isn't it ?